content/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-python-and-powershell.md

```diff
@@ -0,0 +1,242 @@
+
+```python
+# Store our Compute Ops Manager API endpoint
+endpoint = "https://us-west2-api.compute.cloud.hpe.com"
+# Add a header for authorization, using the Access Token generated in task 2	
+headers = {"Authorization": "Bearer " + "paste_access_token_here"}
+```
+
+![figure8](/img/greenlake-com-ops-api-pyt-pow5.png)
+
+I can now send my first request:
+
+```python
+response = requests.get(url=endpoint + '/compute-ops/v1beta2/servers?limit=2', headers=headers)
+```
+
+Once again, I check if the request was successful, the response status code should display 200:
+
+```python
+print(response.status_code)
+```
+
+In order to display the JSON data returned by the request in a human-readable form:
+
+```python
+print(json.dumps(response.json(), indent=2))
+```
+
+![figure 9](/img/greenlake-com-ops-api-pyt-pow6.png)
+
+I see the request returned 2 objects **(count : 2)** and the actual server details are in the **items** array property.
+
+The API supports several query parameters depending on the resource type, such as filter, limit (maximum number of records to return), offset (resource offset to start the response from) and sort (order in which to return the resources in the collection).
+
+Filters provide the ability to limit the resources that take part in the action of a REST call. When a REST call includes a filter, the GET or DELETE action is restricted to a response that meets the filter requirements. Filters are specified by using the query parameter 'filter'.
+
+A simple example of filtering follows:
+
+**GET <URI>?filter=powerState eq 'Off'**
+
+This example shows a simple filter. The resources returned by the query are limited to results with the attribute powerState and the value Off. 
+
+To use a filter on a nested property name, the '/' separator can be specified as follows:
+
+**GET /compute-ops/v1beta2/servers?filter=hardware/model eq 'ProLiant DL365 Gen10 Plus'**
+
+I can test with the following command:
+
+```python
+response = requests.get(url= endpoint + "/compute-ops/v1beta2/servers?filter=contains(hardware/model,'DL365')", headers=headers)
+```
+
+First check if the request was successful as before
+
+```python
+print(response.status_code)
+```
+
+should display 200
+
+Display the number of objects in the response
+
+```python
+print(response.json()\['count'])
+```
+
+It should display 4 
+
+![figure 10](/img/greenlake-com-ops-api-pyt-pow7.png)
+
+Display the JSON data returned by the request in a human-readable form
+
+```python
+print(json.dumps(response.json(), indent=2))
+```
+
+![figure 11](/img/greenlake-com-ops-api-pyt-pow8.png)
+
+Refer to previous sections for more information on filter query syntax and API references.
+
+## What’s next?
+
+In this blog post, we covered how to get started with HPE GREENLAKE FOR COMPUTE OPS MANAGEMENT REST API explained how to post simple API calls through Python and PowerShell. Some future Workshops-on-Demand will allow you to run experiment these examples live, stay tuned.
```

Comment:
"HPE GreenLake for Compute Ops Management REST API" should not be in ALL CAPS. It also needs the word "and" between it an "explained how to post simple API calls..."

The very last sentence needs a rewrite. Perhaps this would do?
"In some of our future Workshops-on-Demand, you will have the opportunity to run live experiments on these examples. Stay tuned for more details!"

I also recommend you then hotlink to the "Workshops-on-Demand" catalog.